src/action.js

```diff
@@ -81,35 +82,33 @@ async function GetLatest() {
         throw new Error(err)
     })
 
+    ProcessRelease(latestRelease)
+
+    core.info('GetLatestRelease Done')
+}
+
+async function ProcessRelease(release) {
     const {
-        data: { tag_name: latestTag, name: latestName, body: latestBody }
-    } = latestRelease
-    latestAsset = latestRelease.data.assets || ''
-    core.info(
-        format('latest_tag:%s,latest_name:%s,latest_body:%s', latestTag, latestName, latestBody))
-
-    tag = latestTag
-    if (body != '') {
-        body += latestBody != '' ? "\n" + latestBody : ''
-    } else {
-        body = latestBody
-    }
-    if (releaseName != '') {
-        releaseName += latestName != '' ? " " + latestName : ''
-    }
-    core.info('GetLatest Done')
+        data: { tag_name: t, name: n, body: b }
+    } = release
+    releaseAsset = release.data.assets || ''
+    core.info(format('tag:%s, name:%s, body:%s', latestTag, latestName, latestBody))
+
+    tag = t
+    releaseName = n
+    body = b
 }
 
 async function DownloadAssets() {
     core.info('DownloadAssets Start')
     let assetDirPath = path.join('.', 'asset_files')
     await io.mkdirP(assetDirPath).catch(err => core.setFailed(err))
-    for (var i in latestAsset) {
-        core.info('Download ' + latestAsset[i].name)
+    for (var i in releaseAsset) {
```

Comment:
here is that mysterious releaseAsset, but it isn't a boolean anymore...the loop itself surprises me...i is being used as an index into releaseAsset (releaseAsset[i].name)...but we have a for...in loop (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in) ... why not just iterate over the object/array and skip the indexiting?